samples/client/petstore/elixir/test/outer_enum_test.exs

```diff
@@ -15,13 +16,18 @@ defmodule OuterEnumTest do
 
   @tag timeout: :infinity
   test "json_decode/2 with valid JSON" do
-    assert Deserializer.json_decode(@valid_json, EnumTest) ==
-             {:ok,
-              %EnumTest{
-                enum_string: "UPPER",
-                enum_number: 1.1,
-                outerEnum: "placed",
-                outerEnumInteger: 1
-              }}
+    enum_test =
+      JSON.decode!(@valid_json)
+      |> then(fn params -> EnumTest.changeset(%EnumTest{}, params) end)
+      |> Ecto.Changeset.apply_action!(:insert)
+
+    assert enum_test ==
+      %EnumTest{
+        enum_string_required: "lower",
```

Comment:
The test was wrong. We were not validating a required field. Again, one of the goodies we get from Ecto.